e2e/cypress/integration/client-admin/create_user.spec.js

```diff
@@ -0,0 +1,58 @@
+describe('Create User', () => {
+  afterEach(() => cy.visit('/signout'))
+
+  beforeEach(() => {
+    cy.visit('/signout')
+    cy.visit('/createuser')
+  })
+
+  before(async function () {
+    const users = await cy.fixture('users')
+    this.user = users[0]
+  })
+
+  it('Does not create a new user with existing email address', () => {
+    cy.get('form').within(async function () {
```

Comment:
I had to add async here or else form wasn't being completed at all, and was timing out.